### PR TITLE
complete ex5

### PR DESCRIPTION
**Pull request by charlieBdev**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -85,4 +86,34 @@ describe('GET /api/articles/:article_id', () => {
...
+                expect(article).toHaveProperty('article_id'), expect.any(Number)
+                expect(article).toHaveProperty('topic'), expect.any(String)
+                expect(article).toHaveProperty('created_at'), expect.any(String)
+                expect(article).toHaveProperty('votes'), expect.any(Number)
+                expect(article).toHaveProperty('article_img_url'), expect.any(String)
+                expect(article).toHaveProperty('comment_count'), expect.any(Number)
+                expect(article).not.toHaveProperty('body')
+            })
+        })
+    });
```
Comment (Jodiech) [inline __tests__/app.test.js:110]: You could separate this out into its own describe block 👍

File: app/controllers/articles.controller.js [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
     })
     .catch(next)
 }
+
+exports.getAllArticles = (_, res, next) => {
+    selectAllArticles()
+    .then((articles) => {
+        res.status(200).send(articles)
+    })
+}
```
Comment (Jodiech) [inline app/controllers/articles.controller.js:19]: Please include a catch block here❗

File: __tests__/app.test.js [hunk 3]
```diff
@@ -85,4 +86,34 @@ describe('GET /api/articles/:article_id', () => {
...
         })
     });
+});
+describe('GET /api/articles', () => {
+    test('200 should return all articles', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body).toBeSortedBy('created_at', { descending: true })
```
Comment (Jodiech) [inline __tests__/app.test.js:96]: Nice use of the jest-sorted package here! 👍

File: app/controllers/articles.controller.js [hunk 4]
```diff
@@ -1,11 +1,19 @@
-const { selectArticleById } = require("../models/articles.model")
+const { selectArticleById, selectAllArticles } = require("../models/articles.model")
 
 exports.getArticleById = (req, res, next) => {
     const { article_id } = req.params
 
     selectArticleById(article_id)
     .then((article) => {
+        article.created_at = article.created_at
```
Comment (Jodiech) [inline app/controllers/articles.controller.js:8]: What is this for? Is this necessary?

**Review by Jodiech — APPROVED** [review]
Hey 👋 great job on this ticket! I've left some comments for you to consider. No need to submit another PR for it. Happy for you to merge 😄

File: app/controllers/articles.controller.js (hunk 4 above)
Comment (charlieBdev) [reply app/controllers/articles.controller.js:8]: Yeah, sorry forgot about that. I have an issue where created_at in the test-data is like - 1602828180000

But in the output from the controller, it's like 2020-07-09T20:11:00.000Z

I wanted to change it but didn't know how.

**Pull request merged**